docs/hello_nextflow/05_hello_channels.md

```diff
@@ -0,0 +1,558 @@
+# Part 4: Hello Channels
+
+In Part 2, you built a pipeline that was completely linear and processed each sample's data independently of the others. However, in real pipelines, you may need to combine data from multiple samples, or combine different kinds of data. Here we show you how to use channels and channel operators to implement a pipeline with more interesting plumbing.
+
+Specifically, we show you how to implement joint variant calling with GATK, building on the pipeline from Part 2.
+
+!!! note
+
+    Don't worry if you're not familiar with GATK or genomics in general. We'll summarize the necessary concepts as we go, and the workflow implementation principles we demonstrate here apply broadly to any use case that follows a similar pattern.
+
+### Method overview
+
+The GATK variant calling method we used in Part 2 simply generated variant calls per sample. That's fine if you only want to look at the variants from each sample in isolation, but that only yields limited information. It's often more interesting to look at variant calls across multiple samples, and to do so, GATK offers an alternative method called joint variant calling, which we demonstrate here.
+
+Joint variant calling involves generating a special kind of variant output called GVCF (for Genomic VCF) for each sample, then combining the GVCF data from all the samples and finally, running a 'joint genotyping' analysis to produce the final cohort-level VCF containing variant calls calculated based on the information from all samples.
+
+![Joint analysis](img/joint-calling.png)
+
+What's special about a sample's GVCF is that it contains records summarizing sequence data statistics about all positions in the targeted area of the genome, not just the positions where the program found evidence of variation. This is critical for the joint genotyping calculation. _TODO: ADD LINK FOR MORE READING_
+
+The GVCF is produced by GATK HaplotypeCaller, the same tool we used in Part 2, with an additional parameter (`-ERC GVCF`). Combining the GVCFs is done with GATK GenomicsDBImport, which combines the per-sample calls into a data store (analogous to a database), then the actual 'joint genotyping' analysis is done with GATK GenotypeGVCFs.
+
+So to recap, we're going to develop a workflow that does the following:
+
+_ADD joint genotyping FLOWCHART_
```

Comment:
```suggestion
```

Reply:
I have another flowchart to add so this needs to stay in